src/protocolsupportpocketstuff/ProtocolSupportPocketStuff.java

```diff
@@ -100,12 +98,11 @@ public void onConnectionHandshake(ConnectionHandshakeEvent e) {
 		if (PocketCon.isPocketConnection(con)) {
 			// = Packet Listeners = \\
 			//con.addPacketListener(new ModalResponsePacket().new decodeHandler(this, con));
-			if (getConfig().getBoolean("hacks.holograms")) { con.addPacketListener(new HologramsPacketListener(con)); }
 			if (getConfig().getBoolean("hacks.player-heads-skins.skull-blocks")) { con.addPacketListener(new SkullTilePacketListener(this, con)); }
 			if (ServerPlatformIdentifier.get() == ServerPlatformIdentifier.SPIGOT) {
-				if (getConfig().getBoolean("hacks.itemframes")) {
+				/*if (getConfig().getBoolean("hacks.itemframes")) {
 					con.addPacketListener(new ItemFramesPacketListener(this, con));
-				}
+				}*/
```

Comment:
Just incorporate code that is commented out like this, should we do it? [Yoda asks]

Slightly better would be something like a globalHacksItemFramesDisabled = true, or so.